core/src/main/java/com/github/mizool/core/validation/ConstraintValidators.java

```diff
@@ -0,0 +1,21 @@
+package com.github.mizool.core.validation;
+
+import java.util.function.Predicate;
+
+public class ConstraintValidators
```

Comment:
Annotate with `@UtilityClass`.
Remove `public` to reduce visibility to package level.

Reply:
can't reduce package visibility, otherwise the CheckPast can't access it